roles/config-rh-sso/tasks/configure-rh-sso.yml

```diff
@@ -6,7 +6,7 @@
 
 - name: "Requesting Access Token"
   uri:
-    url: "http://{{ rh_sso_host }}:{{ rh_sso_port }}/auth/realms/master/protocol/openid-connect/token"
+    url: "https://127.0.0.1:8443/auth/realms/master/protocol/openid-connect/token"
```

Comment:
This should stay `rh_sso_host` to allow for remote access

Reply:
I am thinking better about this. This task is not necessary as part of the deployment of Red Hat Single Sign-On, meaning that we can probably create additional realms similarly to the way we create users and groups as we need, rather than doing this during the initial configuration, considering that RH SSO already brings a Master realm by default.
Back when it was implemented, the main intent was to test some of the aspects of the API and we decided to insert it here, however, I think we can move this functionality to a dedicated task in this role instead.